stubs/dateparser/dateparser_scripts/get_cldr_data.pyi

```diff
@@ -0,0 +1,15 @@
+from typing import Any
```

Comment:
Is `dateparser_scripts` even supposed to be a public API? It fails to import for me because of a missing dependency. I wonder whether we should even have stubs for it.

Reply:
After a second watch, indeed `dateparser_scripts` does not seems to be a public API. I'll delete the whole directory.